pipelines/_ingredients/cpPtcLSST.yaml

```diff
@@ -0,0 +1,59 @@
+description: cp_pipe ptc calibration construction.
+tasks:
+  cpPtcIsr:
+    class: lsst.ip.isr.IsrTaskLSST
+    config:
+      connections.ccdExposure: "raw"
+      connections.outputExposure: "cpPtcIsrExp"
+      expectWcs: false
+      doDefect: true
+      doBias: false
+      doDark: false
+      doFlat: false
+      doDiffNonLinearCorrection: false
+      doBootstrap: true
+      doDeferredCharge: false
+      doBrighterFatter: false
+      doLinearize: true
+      doCorrectGains: false
+      doApplyGains: false
+      doVariance: true
+      doSaturation: false
```

Comment:
I think I also have masking questions.  Is the assumption that we only have saturation from hot pixels, and that those will have already been added to the defect mask, so we can skip saturation here (as there will should not be any astronomical saturated sources)?

Reply:
The first assumption is that until we do the PTC analysis we don't know what the saturation level is so it must be False.  Beyond that, we are assuming that there are no astronomical saturated pixels, yes, and any instrumental saturated pixels will be found by the defect pipeline.